get-new-tool-versions/parsers/verify-added-to-image/verify-python-parser.psm1

```diff
@@ -0,0 +1,21 @@
+function Search-PythonVersionsNotOnImage {
+    param (
+        [string]$ToolName,
+        [string]$ReleasesUrl,
+        [string]$FilterParameter,
+        [string]$FilterArch
+    )
+    
+    $stableReleases = (Invoke-RestMethod $ReleasesUrl) |
+        Where-Object stable -eq $true |
+        ForEach-Object {$_.$FilterParameter.split(".")[0,1] -join"."} |
+        Select-Object -Unique
+    $toolsetUrl = "https://raw.githubusercontent.com/actions/virtual-environments/main/images/win/toolsets/toolset-2019.json"
+    $latestExistingMinorVesion = ((Invoke-RestMethod $toolsetUrl).toolcache |
+        Where-Object {$_.name -eq $ToolName -and $_.arch -eq $FilterArch}).versions |
+        ForEach-Object {$_.split(".")[0,1] -join"."} |
+        Select-Object -Last 1
```

Comment:
suggest rewriting it to use two variables for readability and take the latest one from the start
```suggestion
    $latestVersion = ((Invoke-RestMethod $toolsetUrl).toolcache | Where-Object {$_.name -eq $ToolName -and $_.arch -eq $FilterArch}).versions | Select-Object -Last 1
    $latestMinorVesion = $LatestVersion.Split(".")[1]
```

Reply:
Agree. Updated